packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -524,6 +526,18 @@ export default {
             this.getUserNote();
         },
 
+        /**
+         * Update address lines if localStorage is populated.
+         *
+         * */
+        getAddressFromLocalStorage () {
```

Comment:
Nit: could we please rename this to `loadAddressFromLocalStorage` so it's different from the `addressService`'s name and it also matches our naming convention for those initial `loadXXXXXXX` functions?

Reply:
Fixed.